src/components/orderItem.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import { connect } from "react-redux";
+import Dropdown from './dropdown'
+import Options from '../components/options'
+import { returnExchangeDetails } from '../store/actions'
 import OptionForm from './dropdown'
 
 class OrderItem extends React.Component {
-
     constructor(props) {
         super(props);
+          
+    }
+    returnExchangeStart(choice, e) {
+        this.props.returnExchangeDetails(choice)
     }
-
     render() {
+        var allowedQty = [this.props.item.quantity_return_allowed];
```

Comment:
Should this be an array?

```suggestion
        var allowedQty = this.props.item.quantity_return_allowed;
```